### PR TITLE
Pedro/cow 284 milestone review 2

### PR DESCRIPTION
**Pull request by yvesfracari**

### CODE REVIEW

File: apps/cow-amm-deployer/src/contexts/transactionManagerContext.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}: {
+  children: React.ReactNode;
+}) => {
+  const managedTransaction = useManagedTransaction();
+
+  return (
+    <TransactionManagerContext.Provider
+      value={{
+        managedTransaction,
+      }}
```
Comment (ribeirojose) [inline apps/cow-amm-deployer/src/contexts/transactionManagerContext.tsx:25]: any reason why encapsulate this in another object?

File: apps/cow-amm-deployer/src/contexts/ammDataContext.tsx [hunk 2]
```diff
@@ -47,6 +54,19 @@ export const AmmDataContextProvider = ({
...
 
+  useEffect(() => {
+    if (isPonderAPIUpToDate) {
+      mutateAmm();
+      setIsTransactionAwaiting(false);
+    }
+  }, [isPonderAPIUpToDate]);
+
+  useEffect(() => {
+    if (!["final", "confirmed", "error", "idle"].includes(status || "")) {
```
Comment (ribeirojose) [inline apps/cow-amm-deployer/src/contexts/ammDataContext.tsx:65]: we should remove `confirmed` from this now right?

File: apps/cow-amm-deployer/src/components/DepositForm.tsx [hunk 3]
```diff
@@ -153,10 +147,8 @@ export function DepositForm({
         )}
 
       <Button
-        loading={
-          isSubmitting ||
-          !["final", "idle", "confirmed", "error"].includes(status || "")
-        }
+        loadingText="Depositing..."
+        loading={isSubmitting || isAmmUpdating}
```
Comment (ribeirojose) [inline apps/cow-amm-deployer/src/components/DepositForm.tsx:151]: so are we ignoring the tx now 🤔 ?

Comment (yvesfracari) [reply apps/cow-amm-deployer/src/components/DepositForm.tsx:143]: actually the `isAmmUpdating` is `true` when the status is something being processed.

File: apps/cow-amm-deployer/src/contexts/ammDataContext.tsx (hunk 2 above)
Comment (yvesfracari) [reply apps/cow-amm-deployer/src/contexts/ammDataContext.tsx]: We're using more the ponder update as a trigger, so I think that is ok to let this

**Pull request merged**